docs/source/development/CDFs/cdf_requirements.rst

```diff
@@ -0,0 +1,343 @@
+#####################
+CDF File Requirements
+#####################
+
+The following page describes the necessary contents inside of a CDF file generated for the IMAP mission.
+
+For a more general introduction to CDF files, see :ref:`CDF Introduction`.
+
+For sample code that will generate a file that matches the below guidelines, see :ref:`IMAP xarray_to_cdf Example`
+
+******************************
+Global Attributes
+******************************
+
+Global attributes are used to provide information about the data set as an entity. Together with variables and variable attributes, the global attributes make the data correctly and independently usable by someone not connected with the instrument team, and hence, a good archive product.
+
+Global attributes that have been identified for use with IMAP data products are listed below. Additional Global attributes can be defined but they must start with a letter and can otherwise contain letters, numbers and the underscore character (no other special characters allowed). Note that CDF attributes are case-sensitive and must exactly follow what is shown here.
+
+ISTP Compliant Global Attributes are listed here: `https://spdf.gsfc.nasa.gov/istp_guide/gattributes.html <https://spdf.gsfc.nasa.gov/istp_guide/gattributes.html>`_, and notes about how they are used on IMAP are below -
+
+Data_type
+==========
+This attribute is used by CDF file writing software to create a filename. It is a combination of the following filename components: mode, data level, and optional data product descriptor.
+
+Data_version
+================
+This attribute identifies the version of a particular CDF data file.
+
+Descriptor
+================
+This attribute identifies the name of the instrument or sensor that collected the data.  Both a long name and a short name are given.  For any data file, only a single value is allowed.
+
+For IMAP, one of the following must be used -
+
+* IDEX>Interstellar Dust Experiment
+* SWE>Solar Wind Electrons
+* SWAPI>Solar wind and Pickup Ions
+* CoDICE>Compact Dual Ion Compoition Experiment
+* MAG>Magnetometer
+* HIT>High-energy Ion Teleccope
+* GLOWS>GLObal Solar Wind Structure
+* IMAP-Hi>Interstellar Mapping and Acceleration Probe High
+* IMAP-Lo>Interstellar Mapping and Acceleration Probe Low
+* IMAP-Ultra>Interstellar Mapping and Acceleration Probe Ultra
+
+Discipline
+================
+For IMAP, this value should always be “Space Physics>Heliospheric Physics.”. This attribute describes both the science discipline and sub discipline.
+
+Generation_date
+================
+Date stamps the creation of the file using the syntax yyyymmdd, e.g. 20150923.
+
+Instrument_type
+================
```

Comment:
```suggestion
Instrument_type
===============
```

Reply:
Looks like there are several more of these below, so I will stop commenting on them individually 😅